app/services/bulk_upload/processor.rb

```diff
@@ -3,12 +3,12 @@ class BulkUpload::Processor
 
   BLANK_TEMPLATE_ERRORS = [
     I18n.t("activemodel.errors.models.bulk_upload/lettings/validator.attributes.base.blank_file"),
-    I18n.t("activemodel.errors.models.bulk_upload/sales/validator.attributes.base.blank_file"),
+    I18n.t("validations.sales.2024.bulk_upload.blank_file"),
   ].freeze
 
   WRONG_TEMPLATE_ERRORS = [
     *I18n.t("activemodel.errors.models.bulk_upload/lettings/validator.attributes.base", default: {}).values,
-    *I18n.t("activemodel.errors.models.bulk_upload/sales/validator.attributes.base", default: {}).values,
+    *I18n.t("validations.sales.2024", default: {}).values,
```

Comment:
Does this "validations.sales.2024" need to be "validations.sales.2024.bulk_upload"? Will this store all the translations not just the wrong template errors?

Reply:
Would it make sense to put blank_file; over_max_column_count; no_headers; wrong_field_numbers_count: and wrong_template: under another key under bulk upload e.g. `validations.sales.2024.bulk_upload.upload_errors.blank_file`

Reply:
All very good suggestions, added now!